psi4/src/psi4/libmints/matrix.cc

```diff
@@ -3037,7 +3036,7 @@ void Matrix::load(psi::PSIO *const psio, size_t fileno, SaveType st) {
             psio->read_entry(fileno, name_.c_str(), (char *)fullblock[0], sizeof(double) * sizer * sizec);
 
         set(fullblock);
-        linalg::detail::free(fullblock);
```

Comment:
Why the change?

Reply:
LGTM was reporting a mismatch error. I didn't dig into the innards of `linalg::detail::free`, but `freeblock` seems to be the standard way to clear memory allocated with `to_block_matrix`.